src/lib/bootloader/systeminfo.rb

```diff
@@ -139,10 +139,9 @@ def zipl_device
       #
       # @return [Boolean] true if device is a SCSI device
       def scsi?(device)
-        # in lack of a better idea: check if device name starts with 'sd'
-        # alternatively: device.udev_ids.any?(/^scsi-/)
-        # or: device.udev_paths.any?(/-zfcp-/)
-        device.name.start_with?("/dev/sd")
+        # checking if device name starts with 'sd' is not enough: it could
+        # be a device mapper target (e.g. multipath)
```

Comment:
please add link to bugzilla here to get some context. Otherwise change looks reasonable for me.